packages/coins/src/evm/erc20.json

```diff
@@ -6746,7 +6831,7 @@
     "platforms": {
       "binance": {
         "contract_address": "0xaFF9084f2374585879e8B434C399E29E80ccE635",
-        "decimal_place": 8
+        "decimal_place": 18
```

Comment:
this change seems wrong. Refer: https://bscscan.com/address/0xaFF9084f2374585879e8B434C399E29E80ccE635#readContract#F3